microsoft-edge/webview2/how-to/debug-visual-studio-code.md

```diff
@@ -219,6 +219,24 @@ If you're debugging Office Add-ins, open the add-in source code in a separate in
    ![Run and Debug](./debug-visual-studio-code-images/attach-uwp.png)
 
 
+<!-- ====================================================================== -->
+## Source maps with the `WebResourceRequested` event or virtual host name mapping
```

Comment:
global in PR: 
* Source maps with the `WebResourceRequested` event or virtual host name mapping
* Source maps by using the `WebResourceRequested` event or virtual host name mapping
* Loading source maps by using the `WebResourceRequested` event or virtual host name mapping

https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with

@captainbrosset @dhveerap - I have to decide whether to change 'with' to 'by using', per corp style guide, "with" is an ambig. wildcard word, like "involving", or "having something to do with".